templates/resources/cluster_aks.md.tmpl

```diff
@@ -88,5 +88,22 @@ resource "spectrocloud_cluster_aks" "cluster" {
 }
 ```
 
+## Import
+
+In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import)
+to import {{ .Name }} using `id` with `context` separated by colon. For example:
+
+```terraform
+import {
+  to = {{ .Name }}.example
+  id = "example_id:context"
```

Comment:
Should we add `project` or `tenant` as an example? We do it below for the command, so it's super helpful if we show both.

Reply:
@karl-cardenas-coding changed that too.